prototype2/js/model.js

```diff
@@ -51,14 +51,14 @@ class Model {
         let max = minmax[1];
         let diff = max - min;
         return function (d) {
-            return d > (min + diff * 7.0 / 8.0) ? '#800026' :
-                   d > (min + diff * 6.0 / 8.0) ? '#BD0026' :
-                   d > (min + diff * 5.0 / 8.0) ? '#E31A1C' :
-                   d > (min + diff * 4.0 / 8.0) ? '#FC4E2A' :
-                   d > (min + diff * 3.0 / 8.0) ? '#FD8D3C' :
-                   d > (min + diff * 2.0 / 8.0) ? '#FEB24C' :
-                   d > (min + diff * 1.0 / 8.0) ? '#FED976' :
-                                                  '#FFEDA0';
+            return d > (min + diff * 7.0 / 8.0) ? colors[7] :
+                   d > (min + diff * 6.0 / 8.0) ? colors[6] :
+                   d > (min + diff * 5.0 / 8.0) ? colors[5] :
+                   d > (min + diff * 4.0 / 8.0) ? colors[4] :
+                   d > (min + diff * 3.0 / 8.0) ? colors[3] :
+                   d > (min + diff * 2.0 / 8.0) ? colors[2] :
+                   d > (min + diff * 1.0 / 8.0) ? colors[1] :
+                                                  colors[0];

```

Comment:
Can you update this to a single expression? I understand how it relates to the old code, but now that you're using indices, you don't need this big nested ternary anymore.